test/residual/arrayExpression.js

```diff
@@ -0,0 +1 @@
+[1, "2", [3], ...[4, 5]]
```

Comment:
It is a little bit too early to try and run test262 through the partial evaluator, but yet, that would make sense.